src/draft/NSG/Agent/LegalEntity/NonListedCompany/Establishment/Write.py

```diff
@@ -0,0 +1,1633 @@
+        example="9383",
+        max_length=10,
+    )
+    admin_unit_level_1: Optional[ISO_3166_1_Alpha_3] = Field(
+        None,
+        title="Admin unit level 1",
+        description="The name of the uppermost level of the address, almost always a country. ISO 3166 three "
+        "character (Alpha 3) format.",
+        example=ISO_3166_1_Alpha_3.USA,
+    )
+    admin_unit_level_2: Optional[str] = Field(
+        None,
+        title="Admin unit level 2",
+        description="The name of a secondary level/region of the address, usually a county, state or other such area "
+        "that typically encompasses several localities. Values could be a region or province, "
+        "more granular than level 1.",
+        example="Lapland",
+        max_length=40,
+    )
+
+
+class ManagingDirectors(CamelCaseModel):
+    role: ManagingDirectorRole = Field(
+        ...,
+        title="List of director roles",
+        description="The role of the director",
+        example=ManagingDirectorRole.DIRECTOR,
+    )
+    given_name: str = Field(
+        ...,
+        title="Given name",
+        description="The first name that the person is being called by",
+        example="Mary",
+        max_length=250,
+    )
+    middle_names: str = Field(
+        ...,
+        title="Middle name",
+        description="All the middle names of the person",
+        example="Juliet Olive",
+        max_length=250,
+    )
+    last_name: str = Field(
+        ...,
+        title="Last name",
+        description="The person's current family name",
+        example="Deo",
+        max_length=250,
+    )
+    date_of_birth: date = Field(
+        ...,
+        title="Date of birth",
+        description="The birth day of a person",
+        example=date(1976, 4, 16),
+    )
+    nationality: ISO_3166_1_Alpha_3 = Field(
+        ...,
+        title="Nationality",
+        description="The nationality of a person",
+        example=ISO_3166_1_Alpha_3.USA,
+    )
+
+
+class BoardMembers(CamelCaseModel):
+    role: BoardMemberRole = Field(
+        ...,
+        title="Role",
+        description="The role of the person in the board",
+        example=BoardMemberRole.MEMBER,
+    )
+    given_name: str = Field(
+        ...,
+        title="Given name",
+        description="The first name that the person is being called by",
+        example="Mary",
+        max_length=250,
+    )
+    middle_names: str = Field(
+        ...,
+        title="Middle names",
+        description="All the middle names of the person",
+        example="Juliet Olive",
+        max_length=250,
+    )
+    last_name: str = Field(
+        ...,
+        title="Last name",
+        description="The person's current family name",
+        example="Deo",
+        max_length=250,
+    )
+    date_of_birth: date = Field(
+        ...,
+        title="Date of birth",
+        description="The birth day of a person",
```

Comment:
@hsuomi: Should this be `of *the* person`? Currently exactly as in the spreadsheet.